apps/rush-lib/src/data/test/repo/rush.json

```diff
@@ -17,6 +17,12 @@
     "sampleEmail": "mrexample@contoso.com"
   },
 
+  "rushHooks": {
```

Comment:
>rushHooks [](start = 3, length = 9)

Why is it "rushHooks"?  We're already in the rush.json config file.

(I am sorry to hassle you about this, but you are using a telemetry PR to introduce a general event handling mechanism into Rush without a spec or design discussion.  So it's inevitable that the code review will be a little bumpy.)

Thinking about the name...  maybe the feature should be called "eventHooks" (or "eventScripts"?), and then the "postBuild" event should be "afterRushBuild".  That would allow us to introduce non-Rush-specific hooks later, e.g. "afterNpmInstall".

@iclanton @nickpape-msft